keyboards/handwired/neodox/info.json

```diff
@@ -0,0 +1,133 @@
+{
+    "manufacturer": "MatteoDM",
+    "keyboard_name": "Neodox",
+    "maintainer": "MatteoDM",
+    "backlight": {
+        "driver": "pwm",
+        "levels": 10,
+        "pin": "GP28"
+    },
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "quantum_painter": true,
+        "rgblight": true,
+        "wpm": true
+    },
+    "matrix_pins": {
+        "cols": ["GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP8"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "led_count": 15
+    },
+    "split": {
+        "enabled": true,
+        "main": "matrix_grid",
+        "matrix_grid": ["GP5", "GP15"],
```

Comment:
currently, at time of writing, this setting in info.json doesn't function, but thankfully you've defined it in `config.h`
```suggestion
```